src/shared/components/data-table/components/table-controls/components/rows-per-page-selector/RowsPerPageSelector.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useContext } from 'react';
+import SimpleSelect from 'src/shared/components/simple-select/SimpleSelect';
+import type { RowsPerPage } from 'src/shared/components/data-table/dataTableTypes';
```

Comment:
Import is in the wrong place and also this should be a type import

Reply:
Updated it